### PR TITLE
Add --limit option

### PR DESCRIPTION
**Pull request by nedSaf**
#11

Added a `--limit` option to the csv2sql command.

Applying this option with `--limit=2`:
selection_109

Also tested without the option and it converts the whole file.

@amitaibu PR is ready for code review.

### CODE REVIEW

File: csv2sql.drush.inc [hunk 1]
```diff
@@ -37,6 +38,8 @@ function csv2sql_drush_command() {
 function drush_csv2sql($csv_path) {
   $prefix = drush_get_option('prefix', '_raw');
 
+  $limit = drush_get_option('limit', FALSE);
```
Comment (amitaibu) [inline csv2sql.drush.inc]: validate limit is a numeric value.

File: csv2sql.drush.inc [hunk 2]
```diff
@@ -49,8 +52,9 @@ function drush_csv2sql($csv_path) {
   drush_print(format_string('Starting to convert file "@file" to table "@table".', array('@file' => $path_info['filename'], '@table' => $table_name)));
 
   if (($handle = fopen($csv_path, 'r')) !== FALSE) {
+    $convert_count = 0;
```
Comment (amitaibu) [inline csv2sql.drush.inc]: convert_count => row_number

**Comment by nedSaf** [comment]
@amitaibu PR is ready

File: csv2sql.drush.inc [hunk 3]
```diff
@@ -37,6 +38,13 @@ function csv2sql_drush_command() {
 function drush_csv2sql($csv_path) {
   $prefix = drush_get_option('prefix', '_raw');
 
+  $limit = drush_get_option('limit', FALSE);
+
+  if ($limit && !is_numeric($limit)) {
```
Comment (amitaibu) [inline csv2sql.drush.inc]: this allows passing 1.5
See example from [RESTful](https://github.com/RESTful-Drupal/restful/blob/24fe14f03a40564ac419e117d939f8074766bc78/plugins/restful/RestfulBase.php#L945-L947)

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**